webroot/css/stats/users_languages.css

```diff
@@ -42,6 +42,9 @@
 .usersLanguagesLegend .languageLevel .level {
     background-color: #06A;
     height: 100%;
+    font-size: 12px;
+    float: left;
+    position: relative;
```

Comment:
Instead of setting the font-size, float and position, just adding `display: block` might solve everything.

The misalignment seems to be because of the `level` div being set with a display `inline-block` and for some reason, inline-block messes up the vertical alignment.

Reply:
> The misalignment seems to be because of the `level` div being set with a display `inline-block` and for some reason, inline-block messes up the vertical alignment.

Well, your comment motivated me to look for the reason why this happens and the real problem lies somewhere else. :grin:   

There's another CSS declaration block for the `.languageLevel .level` selector:
https://github.com/Tatoeba/tatoeba2/blob/0ac8959db06237f110834e9aa0b0a3bef1f1dd65/webroot/css/layouts/elements.css#L1707-L1719
As you can see the second inner `div` block is also set to `inline-block`. That's why setting `display:block` helps and the inner `div` behaves as it should.

I've searched through the whole codebase to find the places where we use elements with class `languageLevel` and I've found the following three places:
- https://github.com/Tatoeba/tatoeba2/blob/0ac8959db06237f110834e9aa0b0a3bef1f1dd65/webroot/js/user/language.ctrl.js#L38-L43
- https://github.com/Tatoeba/tatoeba2/blob/0ac8959db06237f110834e9aa0b0a3bef1f1dd65/src/View/Helper/MembersHelper.php#L102-L117
- https://github.com/Tatoeba/tatoeba2/blob/0ac8959db06237f110834e9aa0b0a3bef1f1dd65/src/View/Helper/LanguagesHelper.php#L468-L472

Only in the third place (i.e. the stats page) we also use the inner `level` class so I suggest to simply remove the conflicting blocks from `elements.css` (the empty rule is unnecessary and `.languageLevel.unknown` isn't used anywhere).